src/controllers/travelControllers.js

```diff
@@ -110,13 +111,31 @@ const getUserTravelStatus = async (req, res) => {
 
 const approveTravelRequest = async (req, res) => {
   try {
-    const updatedTravel = await approveTravel(req.params.travel_id);
+    const updatedTravelObj = await approveTravel(req.params.travel_id);
+    const updatedTravel = updatedTravelObj[1][0];
+
+    const notificationObj = await createNotification({
+      recipient_id: updatedTravel.user_id,
+      travel_id: updatedTravel.id,
+      message:
+        `Your travel request created on ${updatedTravel.createdAt} with ${updatedTravel.origin} as your origin has been approved.`,
+    });
+    const notificationMessage = notificationObj.dataValues;
+
+    delete notificationMessage.recipient_id;
+    notificationMessage.title = 'Approved Travel Request.';
```

Comment:
@chuksjoe Good work

why not put the messages **(notificationMessage.title  and the very long one)** in the` utils/utilsmessages` folder?